helm/prefect-server/values.yaml

```diff
@@ -51,8 +51,8 @@ postgresql:
   # existingSecret configures which secret should be referenced
   # for access to the database. If null and `useSubChart` is
   # enabled, the secret will be generated. If using an external
-  # postgres service, this value should be set to a secret
-  # containing the key `postgresql-password`
+  # postgres service, this value should be set to a name of the
+  # Kubernetes secret containing the key `postgresql-password`
```

Comment:
```suggestion
  # postgres service, this value should be set to the name of a
  # Kubernetes secret containing the key `postgresql-password`
```

Reply:
haha, articles is definitely not my forte, thanks